node/pkg/query/query_test.go

```diff
@@ -45,25 +45,55 @@ var (
 	watcherChainsForTest = []vaa.ChainID{vaa.ChainIDPolygon, vaa.ChainIDBSC}
 )
 
-// createPerChainQueryForTesting creates a per chain query for use in tests. The To and Data fields are meaningless gibberish, not ABI.
-func createPerChainQueryForTesting(
+// createPerChainQueryForEthCall creates a per chain query for an eth_call for use in tests. The To and Data fields are meaningless gibberish, not ABI.
+func createPerChainQueryForEthCall(
 	t *testing.T,
 	chainId vaa.ChainID,
 	block string,
 	numCalls int,
 ) *PerChainQueryRequest {
 	t.Helper()
-	callData := []*EthCallData{}
+	ethCallData := []*EthCallData{}
 	for count := 0; count < numCalls; count++ {
-		callData = append(callData, &EthCallData{
+		ethCallData = append(ethCallData, &EthCallData{
 			To:   []byte(fmt.Sprintf("%-20s", fmt.Sprintf("To for %d:%d", chainId, count))),
 			Data: []byte(fmt.Sprintf("CallData for %d:%d", chainId, count)),
 		})
 	}
 
 	callRequest := &EthCallQueryRequest{
 		BlockId:  block,
-		CallData: callData,
+		CallData: ethCallData,
+	}
+
+	return &PerChainQueryRequest{
+		ChainId: chainId,
+		Query:   callRequest,
+	}
+}
+
+// createPerChainQueryForEthCallByTimestamp creates a per chain query for an eth_call_by_timestamp for use in tests. The To and Data fields are meaningless gibberish, not ABI.
+func createPerChainQueryForEthCallByTimestamp(
+	t *testing.T,
+	chainId vaa.ChainID,
+	targetBlock string,
+	followingBlock string,
+	numCalls int,
+) *PerChainQueryRequest {
+	t.Helper()
+	ethCallData := []*EthCallData{}
+	for count := 0; count < numCalls; count++ {
+		ethCallData = append(ethCallData, &EthCallData{
+			To:   []byte(fmt.Sprintf("%-20s", fmt.Sprintf("To for %d:%d", chainId, count))),
```

Comment:
Just out of curiosity, does anyone see this meaningless gibberish, like a log message or something like that?  If not, why not just leave it blank?